Package.swift

```diff
@@ -4,26 +4,55 @@ import PackageDescription
 let package = Package(
   name: "PovioKit",
   platforms: [
-    .iOS(.v13)
+    .iOS(.v13), .macOS(.v13)
   ],
   products: [
-    .library(name: "PovioKitCore", targets: ["PovioKitCore"]),
-    .library(name: "PovioKitNetworking", targets: ["PovioKitNetworking"]),
-    .library(name: "PovioKitPromise", targets: ["PovioKitPromise"]),
-    .library(name: "PovioKitUI", targets: ["PovioKitUI"])
+    .library(
+      name: "PovioKitCore",
+      targets: ["PovioKitCore"]
+    ),
+    .library(
+      name: "PovioKitUtilities", 
+      targets: ["PovioKitUtilities"]
+    ),
+    .library(
+      name: "PovioKitNetworking", 
+      targets: ["PovioKitNetworking"]
+    ),
+    .library(
+      name: "PovioKitPromise", 
+      targets: ["PovioKitPromise"]
+    ),
+    .library(
+      name: "PovioKitUIKit", 
+      targets: ["PovioKitUIKit"]
+    ),
+    .library(
+      name: "PovioKitSwiftUI", 
+      targets: ["PovioKitSwiftUI"]
+    ),
```

Comment:
I'm fine with naming. What I would change is the folder structure. I would have it like this:
- Sources
  - UI
    - UIKit
    - SwiftUI